tests/ident/test_engine_parser_comet_2020_01_04.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+from pathlib import Path
+
+from unify_idents.engine_parsers.ident.comet_2020_01_4_parser import (
+    Comet_2020_01_4_Parser,
+)
+
+
+def test_engine_parsers_comet_init():
+    input_file = Path(__file__).parent.parent / "data" / "BSA1_comet_2020_01_4.mzid"
+    rt_lookup_path = Path(__file__).parent.parent / "data" / "BSA1_ursgal_lookup.csv.bz2"
```

Comment:
please resolve to set a self._package_dir in conftest.py .. the parent.parent will fall apart as soon data is shifted. Check diz :https://github.com/fu/ursgal2/blob/89437556c89aecfe220b6d43362fa805c4df5893/tests/conftest.py#L8

Reply:
and ofcoz do it for all teh occurrences ... ;)